spec/requests/admin/hearings_spec.rb

```diff
@@ -121,4 +121,19 @@
       expect(response).to redirect_to(edit_admin_hearing_url(hearing))
     end
   end
+
+  describe "GET /add_allocation_decision" do
+    let(:offence) { prosecution_case.defendants.first.offences.first }
+
+    it "creates a new allocation decision" do
+      expect {
+        post add_allocation_decision_admin_hearing_url(hearing, offence), headers: headers
+      }.to change(offence.allocation_decisions, :count).by(1)
+    end
+
+    it "redirects to the edit hearings page" do
+      post add_allocation_decision_admin_hearing_url(hearing, offence), headers: headers
+      expect(response).to redirect_to(edit_admin_hearing_url(hearing))
+    end
+  end
 end
```

Comment:
Wondering if we need to beef up the views specs a bit. Now that we're adding fields, we need to see that they are showing correctly